dnssec.c

```diff
@@ -334,6 +337,7 @@ uint16_t ldns_calc_keytag_raw(const uint8_t* key, size_t keysize)
 
 #ifdef HAVE_SSL
 #ifdef USE_DSA
+#ifndef OPENSSL_NO_DEPRECATED_3_0
```

Comment:
Could we perhaps use ``#if OPENSSL_API_COMPAT >= 30000`` instead of extra define?

Reply:
Please change also *configure.ac* and remove/update ``CFLAGS="-DOPENSSL_API_COMPAT=10100 $CFLAGS"`` to new API. May it use new configure parameter ``--without-openssl-compat``?

Reply:
Oh, that is define from OpenSSL 3 itself. Then okay, it might already be set via OPENSSL_API_COMPAT define. Forget this part.